typescript/sdk/src/token/adapters/SealevelTokenAdapter.ts

```diff
@@ -484,6 +502,34 @@ export abstract class SealevelHypTokenAdapter
       this.warpProgramPubKey,
     );
   }
+
+  async getMedianPriorityFee(): Promise<number | undefined> {
+    this.logger.debug('Fetching priority fee history for token transfer');
+
+    const collateralAddress = this.addresses.token;
+    const fees = await this.getProvider().getRecentPrioritizationFees({
```

Comment:
Realizing now that I guess Jito transactions wouldn't show up in this bc they circumvent the typical prioritization fee payment method. This is totally fine, just calling it out